tools/assembly-store-reader/Program.cs

```diff
@@ -50,13 +52,56 @@ static void ShowAssemblyStoreInfo (string storePath)
 				Console.WriteLine ();
 			}
 
+			AssemblyStoreReader? indexStore = null;
+			foreach (var kvp in explorer.Stores) {
+				List<AssemblyStoreReader> storeList = kvp.Value;
+				foreach (AssemblyStoreReader store in storeList) {
+					if (!store.HasGlobalIndex) {
+						continue;
+					}
+
+					indexStore = store;
+					break;
+				}
+			}
+
+			if (indexStore == null) {
+				Console.WriteLine ("Index store not found in the set, not showing hash tables.");
+				return;
+			}
+
+			Console.WriteLine ("Hash tables:");
+			WriteHashTable ("32", indexStore.GlobalIndex32, "x08");
+			WriteHashTable ("64", indexStore.GlobalIndex64, "x016");
+
+			void WriteHashTable (string bitness, List<AssemblyStoreHashEntry> table, string hashFormat)
```

Comment:
Should `bitness` really be a string and not an `int`?  (Crazy town: should it be an enum?)

Reply:
Nah, it's just a label :)